python/GangaLHCb/test/GPI/TestExternalGaudiExec.py

```diff
@@ -77,6 +77,11 @@ def testPrepareJob(self):
 
         from Ganga.GPI import Job, LocalFile, prepareGaudiExec
 
+        import shutil
+        import os
+        if os.path.exists(TestExternalGaudiExec.tmpdir_release):
+            os.system("rm -rf %s/*" % TestExternalGaudiExec.tmpdir_release)
+            
```

Comment:
The below `prepareGaudiExec` fails because there is already a test/release area created by a previous test. This cleans it out.